ceph/cephfs/cephfs_provisioner/cephfs_provisioner.py

```diff
@@ -19,6 +19,7 @@
 import getopt
 import sys
 import json
+import cephfs
```

Comment:
module cephfs seems not be used and should be removed.

Reply:
yeah. I will remove it.